src/net/tcp.rs

```diff
@@ -683,24 +709,24 @@ impl fmt::Debug for TcpListener {
  *
  */
 
-#[cfg(all(unix, not(target_os = "fuchsia")))]
+#[cfg(any(all(unix, not(target_os = "fuchsia")), target_os = "redox"))]
```

Comment:
I'm wondering if these impls should be moved out of this file in order to maintain the goal of "no cfgs" in this file.